spec/style.scss

```diff
@@ -1,4 +1,12 @@
+@import "../components/commons";
+
 .app {
```

Comment:
Maybe in the future this is going to be changed to an `App` component wrapper that already would have the proper styles. This way instead of importing `styles.scss` and add `app` as a className, the user can just make the app hang from `<App />`. We will change it as soon as we write some layout components.